File: python/DPR.py
class DPR:
    __nama = ""
    __id = ""
    __umur = 0
    __bidang = ""
    __partai = ""

    def __init__(self, nama, id, umur, bidang, partai):
        self.__nama = nama
        self.__id = id
        self.__umur = umur
        self.__bidang = bidang
        self.__partai = partai

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_umur(self):
        return self.__umur

    def set_umur(self, umur):
        self.__umur = umur

    def get_bidang(self):
        return self.__bidang

    def set_bidang(self, bidang):
        self.__bidang = bidang

    def get_partai(self):
        return self.__partai

    def set_partai(self, partai):
        self.__partai = partai

File: java/DPR.java
public class DPR {
    private String nama;
    private int id;
    private int umur;
    private String bidang;
    private String partai;

    public DPR() {
    }

    public DPR(String nama, int id, int umur, String bidang, String partai) {
        this.nama = nama;
        this.id = id;
        this.umur = umur;
        this.bidang = bidang;
        this.partai = partai;
    }

    public String getNama() {
        return this.nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public int getId() {
        return this.id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getUmur() {
        return this.umur;
    }

    public void setUmur(int umur) {
        this.umur = umur;
    }

    public String getBidang() {
        return this.bidang;
    }

    public void setBidang(String bidang) {
        this.bidang = bidang;
    }

    public String getPartai() {
        return this.partai;
    }

    public void setPartai(String partai) {
        this.partai = partai;
    }
}
